### ISSUE TITLE
Please add a command line option to set the delay between checks.

### ISSUE THREAD

**Issue by csilvest** [Exceen/4chan-downloader#49]
Currently set by hard-coding to 20 seconds?  I like it to be 120 seconds, so I make an edit to my local file.

Cheers!

**Comment by Exceen** [comment]
there's not hardcoded timeout used right now, it's just the default setting of the urlllib library. you can manually edit line 46

`return urllib.request.urlopen(req).read()`

to 

`return urllib.request.urlopen(req, timeout=120).read()`

if you want the timeout to be 120 seconds

**Comment by Exceen** [comment]
just edit line 120

`time.sleep(20)`

and edit "20" to how many seconds you'd like

what the reason behind this?

**Comment by coavins** [comment]
I agree with this request - 20 seconds is pretty aggressive for a thread I'm going to be watching for days.
